0.data-download/scripts/data_loader.py

```diff
@@ -21,41 +21,40 @@ def load_data(data_directory, adult_or_pediatric="all"):
     )
     model_df = pd.read_csv(model_file, index_col=0)
 
-    # rearrange model info and gene dependency dataframe indices so DepMap_IDs are in alphabetical order
+    # rearrange model info and gene dependency dataframe indices so ModelIDs are in alphabetical order
```

Comment:
You might consider making this a function argument defaulting to `ModelID` to avoid having to change this if the variable name changes again in the future

Reply:
That's a great idea, adding that now